lib/features/email/domain/usecases/get_email_content_interactor.dart

```diff
@@ -28,6 +31,14 @@ class GetEmailContentInteractor {
   ) async* {
     try {
       yield Right<Failure, Success>(GetEmailContentLoading());
+
+      final isDetailEmailCacheExisted = await _openedEmailCacheManager.isOpenedDetailEmailCached(accountId, session.username, emailId);
+
+      if (isDetailEmailCacheExisted && !BuildUtils.isWeb) {
+        yield* _getOpenedEmailCache(session, accountId, emailId, baseDownloadUrl, composeEmail: composeEmail, draftsEmail: draftsEmail);
```

Comment:
We should call `getEmailContent` from jmap server when cache data is not available or corrupted